src/js/api/global-api.js

```diff
@@ -53,6 +53,9 @@ define([
     var _addPlayer = function (api) {
         _uniqueIndex++;
         api.uniqueId = _uniqueIndex;
+        api.isFirstPlayer = function() {
+            return this === _instances[0];
+        };
```

Comment:
@robwalch Let me know if adding this is cool. Can't require `global-api.js` because it would cause a circular dependency:

`controller/controller` => `api/global-api` => `api/api` => `controller/controller`